docs/reference/models/vllm.md

```diff
@@ -73,7 +73,7 @@ vLLM supports AWQ, GPTQ and SqueezeLLM quantized models:
 ```python
 from outlines import models
 
-model = models.vllm("TheBloke/Llama2-7b-Chat-AWQ", quantization="awq")
+model = models.vllm("TheBloke/Llama-2-7b-Chat-AWQ", quantization="awq")
```

Comment:
```suggestion
model = models.vllm("TheBloke/Llama-2-7B-Chat-AWQ", quantization="awq")
```

We should probably capitalize the B there, it seems like TheBloke has changed the name again. 

https://huggingface.co/TheBloke/Llama-2-7B-Chat-AWQ

Reply:
It redirects to the same place but worth doing anyway.

Reply:
Could you accept the suggestion here? I'd like to avoid future naming issues, since I'm not sure how long huggingface supports redirects.

Reply:
@cpfiffer are you asking me?  I proposed the change because what's in the current docs doesn't work but I don't have merge authorization.

Reply:
Oh, you have to make the change on your branch, I cannot edit yours for you. You should be able to click "accept suggestion" in my first comment.

Reply:
Done

Reply:
Great! Appreciate it 😄